app/helpers/application_helper.rb

```diff
@@ -18,21 +18,19 @@ def html_title(title = nil)
     (full_title + app_name).html_safe
   end
 
-  def order_detail_description(order_detail) # TODO: deprecate in favor of OrderDetailPresenter#description_as_html
+  def order_detail_description(order_detail)
```

Comment:
Not sure if there might be any in school specific engines, which is why I've got the deprecation warnings here.